src/main/java/gyro/google/dns/DnsPolicyAlternativeNameServerConfig.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.dns;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.google.api.services.dns.model.PolicyAlternativeNameServerConfig;
+import com.google.api.services.dns.model.PolicyAlternativeNameServerConfigTargetNameServer;
+import gyro.core.resource.Diffable;
+import gyro.google.Copyable;
+import gyro.google.Requestable;
+
+public class DnsPolicyAlternativeNameServerConfig extends Diffable
+    implements Copyable<PolicyAlternativeNameServerConfig>,
+    Requestable<PolicyAlternativeNameServerConfig> {
+
+    private List<DnsPolicyTargetNameServer> targetNameServer;
+
+    /**
+     * Sets an alternative name server for the associated networks. When specified, all DNS queries are forwarded to a name server that you choose. Names such as .internal are not available when an alternative name server is specified. The value may be ``null``.
+     *
+     * @subresource gyro.google.dns.DnsPolicyTargetNameServer
+     */
+    public List<DnsPolicyTargetNameServer> getTargetNameServer() {
+        return targetNameServer;
```

Comment:
Following convention of how we handle lists. This also helps eliminate null checks when using this getter.

```suggestion
        if (targetNameServer == null) {
            targetNameServer = new ArrayList<>();
        }
        
        return targetNameServer;
```